src/components/PeoplePage.tsx

```diff
@@ -15,15 +51,9 @@ export const PeoplePage = () => {
 
           <div className="column">
             <div className="box table-container">
-              <Loader />
-
-              <p data-cy="peopleLoadingError">Something went wrong</p>
-
-              <p data-cy="noPeopleMessage">There are no people on the server</p>
-
-              <p>There are no people matching the current search criteria</p>
+              {isLoading && <Loader />}
```

Comment:
The loading state is already handled at the beginning of the component with a return statement. Therefore, the check for `isLoading` here is redundant and can be removed to simplify the code.